source/ada/lsp-ada_documents.adb

```diff
@@ -835,10 +837,36 @@ package body LSP.Ada_Documents is
                          (To_LSP_String (DN.P_Relative_Name.Text),
                           Prefix => Prefix)
                      then
+                        Context.Trace.Trace
+                          ("Calling Get_Documentation on Node = "
+                           & Image (BD));
+
                         R.label := To_LSP_String (DN.P_Relative_Name.Text);
                         R.kind := (True, To_Completion_Kind
                                    (Get_Decl_Kind (BD)));
                         R.detail := (True, Compute_Completion_Detail (BD));
+
+                        --  Property_Errors can occur when calling
+                        --  Get_Documentation on unsupported docstrings, so
+                        --  add an exception handler to catch them and recover.
+
+                        declare
```

Comment:
Why nott drop `declare` here?

Reply:
Indeed, forgot to remove it. Thanks for noticing